Dockerfile

```
@@ -1,14 +1,20 @@
-FROM golang:1.14
-
-RUN apt-get update && apt-get -y install wget unzip
-
-WORKDIR /tmp
-RUN wget https://releases.hashicorp.com/terraform/0.12.31/terraform_0.12.31_linux_amd64.zip && unzip terraform_0.12.31_linux_amd64.zip -d /usr/local/bin
-
+FROM golang:alpine AS builder
```

```suggestion
FROM golang:1.14-alpine AS builder
```

Please continue to use the same version of go previously specified... This change would allow golang updates to potentially break us.

---

@ScottSuarez done